soc/intel/intel_adsp/ace/Kconfig.series

```diff
@@ -1,8 +1,9 @@
-# Copyright (c) 2022 Intel Corporation
+# Copyright (c) 2022-2024 Intel Corporation
+#
 # SPDX-License-Identifier: Apache-2.0
 
 config SOC_SERIES_INTEL_ACE
-	bool "Intel ACE"
+	bool
 	select SOC_FAMILY_INTEL_ADSP
 	select XTENSA
```

Comment:
These lines should be in the Kconfig file

Reply:
changed